### DISCUSSION TITLE
nerf spawner per mobs.

### DISCUSSION THREAD

**Discussion by matipoirierg** [PurpurMC/Purpur#1110] [Feature Requests]
Hi, I want to suggest the idea to add an option to nerf only specific spawners, since nerf-mob-spawners from spigot nerf all spawners.

**Comment by Igzak** [comment]
Will be amazing only for natural generated spawners like suggested here 
https://github.com/PaperMC/Paper/issues/2411

**Comment by James-P-Bennett** [comment]
Semi-related: 
https://github.com/PurpurMC/Purpur/discussions/1585
